lib/ramble/docs/tutorials/running_an_experiment.rst

```diff
@@ -0,0 +1,270 @@
+rendered to execute each of the experiments that ramble generates.
+
+You can edit these files directly or with the command ``ramble workspace edit``.
+
+To begin, you should edit the ``ramble.yaml`` file to set up the configuration
+for your experiments. For this tutorial, replace the default yaml text with the
+contents of ``$ramble_root/examples/basic_gromacs_config.yaml``:
+
+.. code-block:: yaml
+
+    ramble:
+      variables:
+        processes_per_node: 16
+        mpi_command: 'mpirun -n {n_ranks} -ppn {processes_per_node}'
+        batch_submit: '{execute_experiment}'
+      applications:
+        gromacs: # Application name
+          workloads:
+            water_gmx50: # Workload name from application
+              experiments:
+                pme_single_rank: # Arbitrary experiment name
+                  variables:
+                    n_ranks: '1'
+                    n_threads: '1'
+                    size: '0003'
+                    type: 'pme'
+                rf_single_rank:
+                  variables:
+                    n_ranks: '1'
+                    n_threads: '1'
+                    size: '0003'
+                    type: 'rf'
+            water_bare:
+              experiments:
+                pme_single_rank:
+                  variables:
+                    n_ranks: '1'
+                    n_threads: '1'
+                    size: '0003'
+                    type: 'pme'
+                rf_single_rank:
+                  variables:
+                    n_ranks: '1'
+                    n_threads: '1'
+                    size: '0003'
+                    type: 'rf'
+      spack:
+        concretized: true
+        packages:
+          gcc9:
+            spack_spec: gcc@9.3.0 target=x86_64
+            compiler_spec: gcc@9.3.0
+          impi2018:
+            spack_spec: intel-mpi@2018.4.274 target=x86_64
+            compiler: gcc9
+          gromacs:
+            spack_spec: gromacs@2021.6
+            compiler: gcc9
+        environments:
+          gromacs:
+            packages:
+            - gromacs
+            - impi2018
+
+The second file you should edit is the ``execute_experiment.tpl`` template file.
+This file contains a template script that will be rendered into an execution
+script for each generated experiment. You can feel free to edit it as you need
+to for your given system, but for this tutorial the default value will work.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Setting Up the Experiments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now that the workspace is configured correctly, you can set up the experiments
+in the active workspace using:
+
+.. code-block:: console
+
+    $ ramble workspace setup
+
+This command will create experiment directories, download and expand input files,
+and install the required software stack (and generate spack environments for
+each workload).
+
+It can take a bit to run depending on if you need to build new software and how
+long the input files take to download.
```

Comment:
Maybe add some info about how the logging works, to point people to the different log files.